appservice/src/SiteWrapper.ts

```diff
@@ -45,6 +47,7 @@ export class SiteWrapper {
     public readonly id: string;
     public readonly defaultHostName: string;
     public readonly isFunctionApp: boolean;
+    public readonly isSsl: boolean;
```

Comment:
Do you think people will want to use "isSsl"? Or should this just be "defaultProtocol" that returns "http://" or "https://"?